.github/actions/generate-test-dbs/action.yml

```diff
@@ -5,8 +5,6 @@ runs:
   steps:
     - name: generate test dbs
       run: |
```

Comment:
shall we add a comment in this action that it's expected that qcodes is installed in editable mode before this action can be run?

Reply:
Nah, the action will fail since there is check inside the generate scripts that raises if this is not the case